modules/lightning_features/lightning_core/tests/src/Kernel/Access/AdministrativeRoleCheckTest.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace Drupal\Tests\lightning_core\Kernel\Access;
+
+use Drupal\Core\Routing\RouteMatch;
+use Drupal\lightning_core\Access\AdministrativeRoleCheck;
+use Drupal\Tests\token\Kernel\KernelTestBase;
```

Comment:
Could we add a comment above that why we extended this class instead of system KernelTestBase?